openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -430,7 +430,7 @@ def initialize(id, scope, name, disconnect)
     breakpoints = @@breakpoints[filename]&.filter { |_, present| present }&.map { |line_number, _| line_number - 1 } # -1 because frontend lines are 0-indexed
     breakpoints ||= []
     OpenC3::Store.publish(["script-api", "running-script-channel:#{@id}"].compact.join(":"),
-                          JSON.generate({ type: :file, filename: @filename, scope: @scope, text: @body.to_utf8, breakpoints: breakpoints }))
+                          JSON.generate({ type: :file, filename: @filename, scope: @scope, text: @body&.to_utf8, breakpoints: breakpoints }))
```

Comment:
You should remove the `&` and fix this at line 429. It looks like we made the choice to return `nil` from `::Script.body` if the file doesn't exist. There are three places in `running_script.rb` which needs to handle this by just raising an exception.

Reply:
Done, on 429, by just raising with explanatory text.  How do I find the other two places?  I'm not seeing anything revelatory in the **script wait** output.

Reply:
Line 1272: `text = ::Script.body(@scope, filename)`

Reply:
Done